Tests/Hmmer/text_30_hmmscan_011.out

```diff
@@ -0,0 +1,1425 @@
+# C:\msys64\scr\byerly\builds\NB\20231025 _\thirdparty\bin\Windows-x64\hmmer\bin\hmmscan :: search sequence(s) against a profile database
+# HMMER 3.1b2 (February 2015); http://hmmer.org/
+# Copyright (C) 2015 Howard Hughes Medical Institute.
+# Freely distributed under the GNU General Public License (GPLv3).
+# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+# query sequence file:             C:\Users\byerly\AppData\Local\Temp\tmpt7psecrp.fasta
+# target HMM database:             C:\msys64\scr\byerly\builds\NB\20231025 _\internal\lib\site-packages\anarci\dat\HMMs\ALL.hmm
```

Comment:
I can see here that we also have a Windows path with space. Although this is handled by a different regex, could we still add an assertion to the test to make sure the value is parsed correctly?

Reply:
absolutely